paper-header-panel.css

```diff
@@ -0,0 +1,106 @@
+/**
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
+The complete set of authors may be found at http://polymer.github.io/AUTHORS
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
+*/
+
+:host {
+  @apply(--layout);
+  @apply(--layout-vertical);
+
+  display: block;
```

Comment:
should remove`display: block` since it is already set to `display: flex`.  Also just `@apply(--layout-vertical)` is enough and can remove `@apply(--layout)`